code/SiteBuilder.ts

```diff
@@ -50,25 +50,25 @@ const buildImportsFile = (filepath, varMap) => {
   Disk.write(
     filepath,
     `importOnly\n` +
-      Object.keys(varMap)
-        .map(key => {
-          let value = varMap[key]
+    Object.keys(varMap)
+      .map(key => {
+        let value = varMap[key]
 
-          if (value.rows)
-            return `replace ${key}
+        if (value.rows)
+          return `replace ${key}
  pipeTable
   ${new TreeNode(value.rows)
-    .toDelimited("|", value.header, false)
-    .replace(/\n/g, "\n  ")}`
+              .toDelimited("|", value.header, false)
+              .replace(/\n/g, "\n  ")}`
 
-          value = value.toString()
+        value = value.toString()
```

Comment:
we are probably missing some prettier config settings. not sure why all these lines were changed

Reply:
yes, that's why the files changed.